tests/testthat/test-utils.R

```diff
@@ -362,3 +362,91 @@ test_that("remove_empty_rows works", {
     remove_empty_rows(my_record_id) %>%
     expect_equal(expected_out)
 })
+
+test_that("is_labelled works", {
+  df <- tibble::tribble(
+    ~"one", ~"two", ~"three",
+    1,        2,      3
+  )
+
+  is_labelled <- is_labelled(df)
+  expect_false(is_labelled)
+
+  labels <- c("One", "Two", "Three")
+  labelled::var_label(df) <- labels
+
+  is_labelled <- is_labelled(df)
+  expect_true(is_labelled)
+})
+
+test_that("make_skimr_labs works", {
```

Comment:
I'm not sure this is what we want. For example, is "Asis Min Length" really a good label? Let's look at example output together and see what can be done. This is also a use case for ChatGPT which came up with nice human readable labels when I tried it out.

Reply:
Sure. When I wrote `make_skimr_labs()` I wanted to keep the output programmatic and tried to avoid any manual assignments just in case `skimr` ever changes their names, but I'm open to whatever.

Reply:
Makes total sense!